recipes/xz_utils/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+            msbuild = MSBuild(self)
+            msbuild.build(
+                'xz_win.sln',
+                targets=[target],
+                build_type=self._effective_msbuild_type(),
+                platforms={'x86': 'Win32', 'x86_64': 'x64'},
+                use_env=False)
+
+    def _build_configure(self):
+        with tools.chdir(self._source_subfolder):
+            args = []
+            env_build = AutoToolsBuildEnvironment(self, win_bash=self._is_mingw_windows)
+            args = ['--disable-doc']
+            if self.settings.os != "Windows" and self.options.fPIC:
+                args.append('--with-pic')
+            if self.options.shared:
+                args.extend(['--disable-static', '--enable-shared'])
+            else:
+                args.extend(['--enable-static', '--disable-shared'])
+            if self.settings.build_type == 'Debug':
+                args.append('--enable-debug')
+            env_build.configure(args=args, build=False)
+            env_build.make()
+            env_build.install()
+
+    def build(self):
+        if self.settings.compiler == 'Visual Studio':
+            self._build_msvc()
+        else:
+            self._build_configure()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        if self.settings.compiler == "Visual Studio":
+            inc_dir = os.path.join(self._source_subfolder, 'src', 'liblzma', 'api')
+            self.copy(pattern="*.h", dst="include", src=inc_dir, keep_path=True)
+            arch = {'x86': 'Win32', 'x86_64': 'x64'}.get(str(self.settings.arch))
+            target = 'liblzma_dll' if self.options.shared else 'liblzma'
+            compiler_version = float(self.settings.compiler.version.value)
+            msvc_version = 'vs2017' if compiler_version >= 15 else 'vs2013'
+            bin_dir = os.path.join(self._source_subfolder, 'windows', msvc_version,
+                                   str(self._effective_msbuild_type()), arch, target)
+            self.copy(pattern="*.lib", dst="lib", src=bin_dir, keep_path=False)
+            if self.options.shared:
+                self.copy(pattern="*.dll", dst="bin", src=bin_dir, keep_path=False)
+
+        # Remove/rename forbidden files/folders in central repository
+        pkg_config_dir = os.path.join(self.package_folder, 'lib', 'pkgconfig')
+        if os.path.exists(pkg_config_dir) and os.path.isdir(pkg_config_dir):
+            shutil.rmtree(pkg_config_dir)
+
+        share_dir = os.path.join(self.package_folder, 'share')
+        if os.path.exists(share_dir) and os.path.isdir(share_dir):
+            shutil.move(share_dir, os.path.join(self.package_folder, 'res'))
+
+    def package_info(self):
+        if not self.options.shared:
+            self.cpp_info.defines.append('LZMA_API_STATIC')
+        if self.settings.os == "Windows":
+            self.cpp_info.libs = ["liblzma"]
```

Comment:
@uilianries what is the motivation to keep the library named `liblzma` on windows, instead of [renaming it to `lzma` as in bincrafters' recipe ](https://github.com/bincrafters/conan-lzma/blob/testing/5.2.4/conanfile.py#L152) ?
I'm asking this because it seems to break the build of ffmpeg on windows (see https://ci.appveyor.com/project/bincrafters/conan-ffmpeg/build/job/xjwtp6hh2p8bpc6k#L987 and https://github.com/FFmpeg/FFmpeg/blob/master/configure#L6194)

Reply:
feel free to submit a PR fixing this. we should stay compatible with ffmpeg and other libraries.